app/purchase/index.tsx

```diff
@@ -45,25 +41,35 @@ const PurchaseScreen = () => {
 
   const chosenVehicle = licencePlate ? getVehicle(licencePlate) : defaultVehicle
 
-  const parkingEnd = addMinutes(new Date(), ticketPriceRequest?.duration ?? 0).toISOString()
+  const parkingEnd = new Date(
+    Date.now() + (ticketPriceRequest?.duration ?? 0) * 60_000,
```

Comment:
I think we have already discussed this but is it not possible to buy a ticket for future parking?

Reply:
No, it isn't possible.